ui/src/App.js

```diff
@@ -107,15 +107,15 @@ class App extends Component {
     }
 
     serializeFilters(searchFilters) {
-        let filterStr = [];
+        let filterList = [];
```

Comment:
filterArr or filterArray

Python has lists but I think these are arrays in js

Reply:
Done